### DISCUSSION TITLE
Too many problems during migration and poor documentation on how to make it work

### DISCUSSION THREAD

**Discussion by cstevanatonehos** [OCA/OpenUpgrade#3827] [Q&A]
I know this is not a bug issue or feature request, but has been days since I am trying to use OpenUpgrade to migrate a database from version 12.0 to 13.0 but with no success.

The documentation is just one page and always different errors keeps popping up when I run the command.
Is there someone working on providing good documentation and help or will it never change?

This may seem a bad critic but it would be benificial to everyone to have a better introduction to Odoo for newcomers.
I am opening this "issue" because I really don't know how to proceed and I feel lost.

I see a lot of logs like "model is declare but cannot be loaded! (Perhaps a module was partially removed or renamed)" and "module: not installable, skipped".
At the end when it seems to load the modules, I always see calls to "/longpolling/poll HTTP/1.1" 200 - 4" without stopping.

**Comment by SirTakobi** [comment]
There is a RFC https://drive.google.com/file/d/1sRis_GoZiVUOfdx-IJA8HMMDgXvhYZzV for improving the documentation (published in https://odoo-community.org/about/rfq-process).
Looking at its schedule it should be done by now, but maybe it just never happened.
